mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -960,6 +962,21 @@ func TestAccComposerEnvironment_fixPyPiPackages(t *testing.T) {
 	})
 }
 
+// This bootstraps the IAM roles needed for the service agents when using encryption.
+func grantEncrypterDecrypterRoleToServiceAgents(t *testing.T) {
```

Comment:
It might make sense to move this piece of work out into a separate PR for 2 reasons (but defer to you if it makes sense to keep it):
1. In case we experience any sort of issues with project permissions when these bootstrap functions are called in the nightly tests, it would make for a smaller rollback and smaller impact.
2. Until we remove all of the places in our examples/tests where these bindings are created, I think they will still be subject to being occasionally deleted (perhaps these were already removed though).

Reply:
Done.